rules/windows/powershell/powershell_classic/posh_pc_renamed_powershell.yml

```
@@ -27,6 +27,8 @@ detection:
             # In some cases powershell was invoked with inverted slashes
             - 'HostApplication=C:/Windows/System32/WindowsPowerShell/v1.0/powershell'
             - 'HostApplication=C:/Windows/SysWOW64/WindowsPowerShell/v1.0/powershell'
+            - 'HostApplication=C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\powershell.exe'
+            - 'HostApplication=C:\\WINDOWS\\SysWOW64\\WindowsPowerShell\\v1.0\\powershell.exe'
```

To avoid confusion with regex escapes. 
```suggestion
            - 'HostApplication=C:\\\\WINDOWS\\\\system32\\\\WindowsPowerShell\\\\v1.0\\\\powershell.exe'
            - 'HostApplication=C:\\\\WINDOWS\\\\SysWOW64\\\\WindowsPowerShell\\\\v1.0\\\\powershell.exe'
```